app/example/router.py

```diff
@@ -25,5 +25,5 @@ async def db_query(db=Depends(get_db)):
 
 @router.get("/http")
 async def http_query(client=Depends(async_client)):
-    resp = await client.get("http://localhost:9999/test")
+    resp = await client.get("http://localstack:4566/health")
```

Comment:
Might be good to pull the URL from config. For most tenants they'd need to override it on a per-environment basis, so showing that pattern here would be good.